src/main/java/com/ldtteam/structurize/network/messages/UpdateAutoScannerMessage.java

```diff
@@ -0,0 +1,101 @@
+package com.ldtteam.structurize.network.messages;
+
+import com.ldtteam.structurize.items.ItemCommandTool;
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraftforge.fml.LogicalSide;
+import net.minecraftforge.network.NetworkEvent;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Updates the command in a command block auto-scanner.
+ */
+public class UpdateAutoScannerMessage implements IMessage

```

Comment:
Better name please, auto scanner is our specific usage, something like UpdateScanCommandBlock would fit better

Reply:
To me "auto scanner" == "scan command block", but ok.